lib/ProductOpener/Packaging.pm

```diff
@@ -475,6 +473,11 @@ sub get_checked_and_taxonomized_packaging_component_data ($tags_lc, $input_packa
 		}
 	}
 
+	# If we don't have data at all, return undef
+	if (not $has_data) {
+		return;
+	}
+
 	return $packaging_ref;
 }
```

Comment:
it's nice you fix bugs, but shan't we add some unit test to this function ?

Reply:
analyze_and_combine_packaging_data() has unit tests in [tests/t/packagings.t](https://github.com/openfoodfacts/openfoodfacts-server/blob/main/tests/unit/packaging.t)

I will add a few more when nothing is recognized.